pkg/authorize/tollbooth/tollbooth.go

```diff
@@ -67,12 +65,10 @@ func (a *authorizer) AuthorizeCluster(token, cluster string) (string, error) {
 
 	response := &clusterRegistration{}
 	if err := json.Unmarshal(body, response); err != nil {
-		level.Warn(a.logger).Log("msg", "upstream server response could not be parsed", "to", a.to)
 		return "", fmt.Errorf("unable to parse response body: %v", err)
 	}
 
 	if len(response.AccountID) == 0 {
-		level.Warn(a.logger).Log("msg", "upstream server responded with an empty user string", "to", a.to)
```

Comment:
do we log these now in the caller?

Reply:
Yes, any error returned gets logged. I wanted to simplify logging a bit. Logs are done at every level here. It is difficult to reason about. When adding a log somewhere, you don't know if it has already been logged in called functions. Now, only the caller of this function do logs.